### PR TITLE
Fix ignored error code for unit tests.

### PR DESCRIPTION
**Pull request by kacf**
Changelog: None

Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

**Comment by kacf** [comment]
Confirmed failing (which it should) for bf534d1e51a398f4f4ccbee2b750d0869fe9dfe5 [here](https://gitlab.com/Northern.tech/Mender/mender/-/pipelines/421733992).

File: client/client_test.go [hunk 1]
```diff
@@ -584,9 +584,11 @@ func TestListSystemCertsFound(t *testing.T) {
 	}
 
 	for name, test := range tests {
-		sysCerts, err := nrOfSystemCertsFound(test.certDir)
-		test.assertFunc(t, err)
-		assert.Equal(t, test.certificatesExpected, sysCerts, name)
+		t.Run(name, func(t *testing.T) {
```
Comment (oleorhagen) [inline client/client_test.go:587]: :+1:

File: client/client.go [hunk 2]
```diff
@@ -327,6 +328,17 @@ func nrOfSystemCertsFound(certDir string) (int, error) {
 		)
 	}
 	for _, certFile := range files {
+		// Need to re-stat here because ReadDir does not resolve
```
Comment (oleorhagen) [inline client/client.go:331]: :ok_hand:

**Review by oleorhagen — APPROVED** [review]
Thanks for fixing this :)

**Comment by kacf** [comment]
Merging without mender-qa pipeline, since this only affects unit tests.

**Pull request merged**